editor/components/colors/utils.js

```diff
@@ -15,7 +16,15 @@ import { find, kebabCase } from 'lodash';
  */
 export const getColorValue = ( colors, namedColor, customColor ) => {
 	if ( namedColor ) {
-		const colorObj = find( colors, { name: namedColor } );
+		let colorObj = find( colors, { slug: namedColor } );
+
+		if ( typeof colorObj === 'undefined' && typeof ( colorObj = find( colors, { name: namedColor } ) ) !== 'undefined' ) {
+			deprecated( 'Using color objects without slugs', {
```

Comment:
Thank you for all the updates @Luehrsen this is looking good!
Maybe can use 'Referencing color by name in attributes when a slug exists.' as the message because we are not deprecating color objects without slugs. Color can still be set without slugs and an automatic slug is computed which is fine for single language sites.

Reply:
+1 for this implementation.

@jorgefilipecosta I agree that in this specific case here your suggestion is more fitting as a message, however I thought we _do_ deprecate color objects without slugs. Of course they remain working, but providing colors without a slug is wrong. Sure it works for single language sites, but we should have multilingual sites or sites where they switch the language at some point in mind and both educate and enforce solid patterns to support these.